### DISCUSSION TITLE
curl: (60) SSL certificate problem: unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by dragfireon** [Cyfrin/foundry-full-course-cu#3066] [Q&A] [answered]
I'm using windows WSL with ubuntu. While running command "curl -L https://foundry.paradigm.xyz/ | bash". I'm getting the below error: 
![ss1](https://github.com/user-attachments/assets/ffabc1f2-3664-4432-8161-d995d8bb29ba)

Things I tried:

a. Runing it using -k/--insecure " "curl -k https://foundry.paradigm.xyz/ | bash"" tags it throws: 
![ss2](https://github.com/user-attachments/assets/270b100b-5569-4e60-9947-2751618d9aba)

b. Added CURL_CA_BUNDLE to env variables with values as path/to/cacert.pem

c. updated the command to curl -cacert path/to/cacert.pem -L https://foundry.paradigm.xyz/ | bash

Please guide me in how I can resolve this issue?

**Comment by EngrPips** [comment, marked as answer]
Hello @dragfireon, Please check this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2518#discussioncomment-10713419  and this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2700#discussioncomment-10986871 and see if it will help you resolve this issue.

**Reply by dragfireon** [reply]
Thanks @EngrPips for the help. I was able to configure everything using above threads.

**Reply by EngrPips** [reply]
Awesome, I am glad those discussions helped you. Keep crushing it.
